kolena/workflow/io.py

```diff
@@ -18,9 +18,9 @@
 
 import pandas as pd
 
+from kolena._utils.datatypes import _DATA_TYPE_MAP
+from kolena._utils.datatypes import DATA_TYPE_FIELD
```

Comment:
Do we want to also move this file up top and re-export them under workflow for backward compatibility?